test/functional/tests/ci/test_basic.py

```diff
@@ -32,3 +35,38 @@ def test_cas_version():
         if splitted_version[1] != file_cas_version:
             TestRun.LOGGER.error(f"""Version of {splitted_version[0]} from cmd doesn't match
              with file. Expected: {file_cas_version} Actual: {splitted_version[1]}""")
+
+
+@pytest.mark.require_disk("cache_1", DiskTypeSet([DiskType.nand, DiskType.optane]))
+def test_negative_start_cache_scenarios():
+    """
+    title: Test starting cache on cache device
+    description:
+      Check for negative cache start scenarios
+    pass criteria:
+      - Does start cache on cache device
+      - Doesn't start cache on the same device with another id
+      - Doesn't start cache on another partition with the same id
+    """
+    with TestRun.step("Set up device"):
+        cache_dev = TestRun.disks["cache_1"]
+        cache_dev.create_partitions([Size(2000, Unit.MebiByte), Size(2000, Unit.MebiByte)])
+        partition_1 = cache_dev.partitions[0]
+        partition_2 = cache_dev.partitions[1]
```

Comment:
We pretty often align to convention:
```
cache_dev = TestRun.disks["cache_1"]
cache_dev.create_partitions([Size(2000, Unit.MebiByte)])
cache_dev = cache_dev.partitions[0]
```
using shadowing variables. So cache_dev is most often name for cache device as partition. Consider in context of test maybe convention of `cache_dev_used` or `cache_dev_occupied` and `cache_dev_free`.